### PR TITLE
[v15] Document the new screen_size field for windows desktops

### PR DESCRIPTION
**Pull request by zmb3**
Backports #38048

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/testplan.md [hunk 1]
```diff
@@ -1378,6 +1382,29 @@ TODO(lxea): replace links with actual docs once merged
...
+
+|   | Tunnel node | Direct dial node |
+|---|---|---|
+| `tsh ssh` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh --no-resume` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh` (from a root cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh --no-resume` (from a root cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| OpenSSH (without `ProxyCommand`) | n/a | <ul><li> [ ] </ul></li> |
+| OpenSSH's `ssh-keyscan` | n/a | <ul><li> [ ] </ul></li> |
+
```
Comment (ibeckermayer) [inline .github/ISSUE_TEMPLATE/testplan.md:1407]: Looks unrelated, do you mean to smuggle this in here too?

File: docs/pages/desktop-access/reference/configuration.mdx [hunk 2]
```diff
@@ -43,3 +43,26 @@ desktop sessions are performed over this tunnel. To enable this mode, ensure
 that `windows_desktop_service.listen_addr` is *unset*, and point
 `teleport.proxy_server` at a Teleport Proxy.
 
+## Screen size
+
+By default, Teleport will set the screen size of the remote desktop session
+based on the size of your browser window. In some cases, you may wish to
+configure specific hosts to use a specific screen size. To do this, set the
+`screen_size` attribute on the `windows_desktop_resource`:
```
Comment (ibeckermayer) [inline docs/pages/desktop-access/reference/configuration.mdx:51]:
```suggestion
`screen_size` attribute on the `windows_desktop` resource:
```

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (zmb3) [reply .github/ISSUE_TEMPLATE/testplan.md:1407]: Yeah, we don't usually backport test plan changes, but since I lumped it in with a docs update in master there was some extra stuff in the backport.

File: docs/pages/desktop-access/reference/configuration.mdx (hunk 2 above)
Comment (zmb3) [reply docs/pages/desktop-access/reference/configuration.mdx:51]: Good catch.

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (probakowski) [reply .github/ISSUE_TEMPLATE/testplan.md:1407]: Do we want to remove it then from this PR?

**Review by probakowski — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/testplan.md (hunk 1 above)
Comment (zmb3) [reply .github/ISSUE_TEMPLATE/testplan.md:1407]: I don't see why we would. If we remove it, any time someone backports a PR that touches the test plan they'll have a merge conflict. If we include it here, future backports will be conflict-free.

**Review by ibeckermayer — APPROVED** [review]
